action.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Jira Check Issue Status'
+description: 'check the Jira issue status'
+author: 'Michael Hakim'
+inputs:
+  organization-name: 
+    description: 'The name of the organization on Jira'
+    required: true
+    default: 'gianteagle'
+  jira-token:  
+    description: 'Jira Token'
+    required: true
+  jira-user:
+    description: 'Jira User'
+    required: false
+runs:
+  using: "composite"
+  steps:
+      - name: First Step
+        shell: bash
+        run: |
+          echo "${GITHUB_HEAD_REF}"
+          echo "hlinfr-152-jirastatus-action" | grep -Eo "[aA-zZ]+-[0-9]+" | tr '[:lower:]' '[:upper:]'
```

Comment:
Did we want to echo this?

Reply:
Ah that was for testing I removed it, thanks for catching that.